packages/babel-helpers/src/helpers.js

```diff
@@ -1,9 +1,11 @@
 import template from "@babel/template";
 
-const helpers = {};
+const helpers = Object.create(null);
 export default helpers;
 
 helpers.typeof = () => template.program.ast`
+  "minVersion: 7.0.0-beta.0";
```

Comment:
imho it's slightly weird to use directive-like for this, making it a property of the helper (or wrapping helpers in objects with this and smth like `create` method) would make it less magical

Reply:
I agree, what about `helpers.typeof.minVersion = 7.0.0-beta.0`?

Reply:
I was being lazy :P Since none of this is actually part of the public API, I figured refactoring it later would be trivial. I'll see if I can make it better now though.